lib/declarations.d.ts

```diff
@@ -384,11 +384,11 @@ interface IPathFilteringService {
 	isFileExcluded(file: string, rules: string[], rootDir: string): boolean
 }
 
-interface ICordovaMigrationService {
-	downloadCordovaMigrationData(): IFuture<void>;
+interface IFrameworkMigrationService {
+	downloadMigrationData(): IFuture<void>;
 	getSupportedVersions(): IFuture<string[]>;
-	pluginsForVersion(version: string): IFuture<string[]>;
-	migratePlugins(plugins: string[], fromVersion: string, toVersion: string): IFuture<string[]>;
+	pluginsForVersion?(version: string): IFuture<string[]>;
```

Comment:
Currently `plugins` is valid only for cordova but this function is part from `IFrameworkMigrationService` so I suggest to rename it to `migrationDataForVersion`